### PR TITLE
fix #267131 - [MusicXML export] print new-system missing in multi-mea…

### PR DESCRIPTION
**Pull request by lvinken**
…sure rests

### CODE REVIEW

**Comment by lvinken** [comment]
Bug affects both 2.x and 3.x, requires merging into 2.2 and master.

File: mscore/exportxml.cpp [hunk 1]
```diff
@@ -4446,6 +4446,25 @@ static void initReverseInstrMap(MxmlReverseInstrumentMap& rim, const MxmlInstrum
...
+ or the enclosing multi-measure rest.
+ */
+
+static System* findSystem(const Measure* const m)
+{
+      auto sys = m->mmRest1()->parent();
+      Q_ASSERT(sys && sys->type() == ElementType::SYSTEM);
+      //qDebug("findSystem(m %p type %hhd '%s')", m, m->type(), m->name());
+      //qDebug("findSystem(sys %p type %hhd '%s')", sys, sys->type(), sys->name());
+      return dynamic_cast<System*>(sys);
```
Comment (lasconic) [inline mscore/exportxml.cpp]: Please use static_cast

Comment (lasconic) [reply mscore/exportxml.cpp]: or better toSystem()

Comment (lasconic) [reply mscore/exportxml.cpp]: What about using just 
```
return m->mmRest1()->system();
```

**Comment by lasconic** [comment]
Merged manually with the suggested changes in master dfb24a397c0eb9f3d5c5e2e606ac14433da68136

**Comment by lasconic** [comment]
and in 2.2 8968e0c58f4b859c390db77ecf1f3d3f1703df40

**Pull request closed without merging**